### DISCUSSION TITLE
Do you have a plan to implement APPLICATION_SYNC group for event notifications?

### DISCUSSION THREAD

**Discussion by ucpr** [pipe-cd/pipecd#4732] [Q&A]
Sorry if this is the wrong category.

Do you have a plan to implement APPLICATION_SYNC group for event notifications?

To avoid differences between the code on GitHub and LiveState, it would be nice to receive event notifications when an application  is Out of Sync. 

https://pipecd.dev/docs-v0.45.x/user-guide/managing-piped/configuring-notifications

**Comment by ffjlabo** [comment]
@ucpr 
Thank you for the question!
Currently, we don't work on it. So feel free to plan :)
Questions are always welcome to plan!
